package.json

```diff
@@ -82,31 +82,31 @@
     "react-native": "0.69.11",
     "react-native-background-timer": "2.4.1",
     "react-native-calendar-events": "2.2.0",
-    "react-native-callstats": "3.73.7",
+    "react-native-callstats": "3.73.22",
     "react-native-default-preference": "1.4.4",
-    "react-native-device-info": "8.4.8",
-    "react-native-dialog": "https://github.com/jitsi/react-native-dialog/releases/download/v9.2.2-jitsi.1/react-native-dialog-9.2.2.tgz",
```

Comment:
Don't, we reply on our patch.

Reply:
I know we did in the past, but I tested and they seem to work as expected.

Reply:
https://github.com/facebook/react-native/issues/31345 
Still there. I will revert this.

Reply:
Placeholder text color, button labels and input written text look as expected on both Android and iOS, light and dark mode.

Reply:
No, it's about the use for the native Modal. We cannot use it as is.